cpp/include/libxtreemfs/file_handle_implementation.h

```diff
@@ -67,12 +68,14 @@ class VoucherManager : public rpc::CallbackInterface<xtreemfs::pbrpc::OSDFinaliz
                  const Options& volume_options,
                  const pbrpc::Auth& auth_bogus,
                  const pbrpc::UserCredentials& user_credentials_bogus);
+  ~VoucherManager();
```

Comment:
Mark destructor as `virtual` to make sure it gets called even when `delete`d as a `rpc::CallbackInterface` pointer.